griptape/schemas/memory/conversation_memory_schema.py

```diff
@@ -8,6 +8,7 @@ class Meta:
 
     type = fields.Str()
     runs = fields.List(fields.Nested(RunSchema()))
+    buffer_size = fields.Int()
```

Comment:
You can just use `fields.Int(allow_none=True)` with Marshmallow. No need for manual `None` handling.